storage/local/mapper.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/prometheus/log"
 
 	clientmodel "github.com/prometheus/client_golang/model"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
nit: non-renamed import goes into a separate import group before the renamed one. (of course we could change that rule at some point if it seems too silly)